websites/api.py

```diff
@@ -204,3 +206,10 @@ def mail_website_admins_on_publish(website: Website, version: str, success: bool
                     "version": version,
                 },
             )
+
+
+def detect_mime_type(uploaded_file: UploadedFile) -> str:
+    """Detect mime type of an uploaded file"""
+    magic = Magic(mime=True)
+    chunk = next(uploaded_file.chunks(chunk_size=2048))
+    return magic.from_buffer(chunk)
```

Comment:
An uploaded JSON file got assigned 'text/plain'  instead of 'application/json', not sure if that's worth worrying about.

Reply:
Looks like it's a known bug that's not easily fixed https://github.com/ahupp/python-magic/issues/193